jenkins-cicd-vmss/azuredeploy.json

```diff
@@ -108,7 +101,7 @@
     }
   },
   "variables": {
-    
+    "VMResourceGroup": "[concat(resourceGroup().name, '_VMSS_', uniqueString(resourceGroup().id))]"
```

Comment:
This can create a RG name that's too long

[concat(take(resourceGroup().name, 70), '_VMSS_', uniqueString(resourceGroup().id))]